wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -504,6 +515,23 @@ async def run(
             kubernetes_asyncio, resource_args
         )
 
+        # If using pvc for code mount, move code there.
+        if launch_project.job_base_image is not None:
+            if SOURCE_CODE_PVC_NAME is None:
+                raise LaunchError(
+                    "SOURCE_CODE_PVC_NAME environment variable not set. "
```

Comment:
Can we make this error more informative? Indicate what users can set in the helm chart to make this not error